### ISSUE TITLE
Batch layer comes after Relu in your vgg implementation fully connected layers. That is not common practice in deep nets.

### ISSUE THREAD

**Issue by gebrahimi91** [mi-lad/snip#2]

**Comment by mil-ad** [comment]
You're right and they do come before ReLUs in the conv layers. There are normally no batchnorms in the classifier but since the paper says they replaced dropout with BN I must have find-replaced all dropout layers with batchnorm. 

I came across [this Reddit thread](https://www.reddit.com/r/MachineLearning/comments/67gonq/d_batch_normalization_before_or_after_relu/) and now I'm not sure where BatchNorm belongs anymore.

**Comment by gebrahimi91** [comment]
https://www.youtube.com/watch?v=tNIpEZLv_eg&t=328s
Watch this video from Andrew Ng. He mentions in third minute he talks about that and mentions that normalizing the values before applying activation function is much more common.
